src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1286,14 +1372,17 @@ impl Cage {
                         "we don't support fstat on sockets yet",
                     );
                 }
+                // Streams don't have inodes, so we'll populate statbuf with dummy info
```

Comment:
a line explaining what the dummy info is would be good for clarification

Reply:
I checked, and there is no particular reason for the dummy info being these values, they could be anything, but probably this is something that was taken from the live environments in the past.